runtime/CMakeLists.txt

```diff
@@ -91,11 +91,15 @@ set(CMAKE_INSTALL_PREFIX "${CMAKE_CURRENT_SOURCE_DIR}" CACHE STRING "" FORCE)
 
 set(OMR_INSTALL_BIN_DIR "${j9vm_SOURCE_DIR}" CACHE PATH "")
 set(OMR_INSTALL_LIB_DIR "${j9vm_SOURCE_DIR}" CACHE PATH "")
-set(OMR_INSTALL_ARCHIVE_DIR "${j9vm_SOURCE_DIR}/lib" CACHE PATH "")
+if(OMR_OS_ZOS)
+	set(OMR_INSTALL_ARCHIVE_DIR "${j9vm_SOURCE_DIR}" CACHE PATH "")
+else()
+	set(OMR_INSTALL_ARCHIVE_DIR "${j9vm_SOURCE_DIR}/lib" CACHE PATH "")
+endif()
```

Comment:
It's not clear how "install" directories within the source tree ever made any sense.

Reply:
I am seeing these referenced in the OMR repo, and bunch of other properties similar like here: https://github.com/eclipse-omr/omr/blob/27a7d57116d3dc82419b7a08955d58bdd479fd41/CMakeLists.txt#L69-L73 

I'm guessing the `ARCHIVE` `LIB` `INC` comes from how it is used, such as here: https://github.com/eclipse-omr/omr/blob/27a7d57116d3dc82419b7a08955d58bdd479fd41/util/a2e/CMakeLists.txt#L52-L56

Is your suggestion/proposal to reword the prefix from `INSTALL` to `SOURCE` ?